blockchain/treasury.go

```diff
@@ -790,6 +790,13 @@ func (b *BlockChain) maxTreasuryExpenditureDCP0007(preTVINode *blockNode) (int64
 //
 // The passed node MUST correspond to a node immediately prior to a TVI block.
 func (b *BlockChain) maxTreasuryExpenditure(preTVINode *blockNode) (int64, error) {
+	isRevertPolicyActive, err := b.isRevertTreasuryPolicyActive(preTVINode)
```

Comment:
I just wanted to note that both the determination of the result of the vote via this function and the expenditure calculation below are for the node _AFTER_ `preTVINode` so they line up correctly.